### PR TITLE
Update documentation-contributions.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added "working with us"

### CODE REVIEW

File: contributors/contributors/ways-to-contribute/documentation-contributions.md [hunk 1]
```diff
@@ -15,3 +15,24 @@ Some additional ideas on where to start contributing to Mattermost product docs:
 3. If you found it challenging to find specific information from the homepage, through the left-hand sidebar, or by searching, that's also excellent feedback for the internal writing team. If you're having trouble finding something, others will too. Together, we can figure out how make that content easier to find.
 4. And one final idea: In GitHub, you'll see a number of Docs issues you can contribute to here: [https://github.com/mattermost/docs/issues](https://github.com/mattermost/docs/issues). Any issue marked as **Help Wanted** and/or **Up For Grabs** is something the writing team would love to incorporate into the product docs. If there's something in that list that appeals to you, please let me know, and we can discuss the request in more detail together.
 
+## Working with us
+
+We really the time taken by contributors to help us, and want to help every contributor submit meaningful content that adds value to our documentation. We refer to these types of contributions as valid contributions.
```
Comment (cwarnermm) [inline contributors/contributors/ways-to-contribute/documentation-contributions.md]: I believe there's a word missing between "really" and "the" in the first sentence.

**Review by cwarnermm — DISMISSED** [review]
Love this! One small missing word suspected and noted inline.

File: contributors/contributors/ways-to-contribute/documentation-contributions.md (hunk 1 above)
Comment (justinegeffen) [reply contributors/contributors/ways-to-contribute/documentation-contributions.md]: You are correct! Thank you. :)

Comment (justinegeffen) [inline contributors/contributors/ways-to-contribute/documentation-contributions.md]:
```suggestion
We really appreciate the time taken by contributors to help us, and want to help every contributor submit meaningful content that adds value to our documentation. We refer to these types of contributions as valid contributions.
```

**Review by cwarnermm — APPROVED** [review]
Thank you!

**Review by cwarnermm — APPROVED** [review]
Love the additions!

**Pull request merged**